test/utils/helpers.ts

```diff
@@ -0,0 +1,48 @@
+import { ethers } from "ethers";
+
+import { randomBN } from "./encoding";
+
+import type {
+  AdvancedOrder,
+  CriteriaResolver,
+  Fulfillment,
+  Order,
+} from "./types";
+
+export const VERSION = `1.1${process.env.REFERENCE ? "-reference" : ""}`;
+
+export const minRandom = (min: ethers.BigNumberish) => randomBN(10).add(min);
+
+export const getCustomRevertSelector = (customErrorString: string) =>
+  ethers.utils
+    .keccak256(ethers.utils.toUtf8Bytes(customErrorString))
+    .slice(0, 10);
+
+export const simulateMatchOrders = async (
+  marketplaceContract: ethers.Contract,
+  orders: Order[],
+  fulfillments: Fulfillment[],
+  caller: ethers.Wallet,
+  value: ethers.BigNumberish
+) => {
+  return marketplaceContract
```

Comment:
nit: can remove `=> {
return ...
}` and just do `=> marketplaceContract...`

(just a minor style thing, your call here)